websites/management/commands/unpublish_sites.py

```diff
@@ -0,0 +1,124 @@
+"""Unpublish course webpages provided to the command that have no dependencies. If there are dependencies, print the dependencies."""
+import json
+
+from django.conf import settings
+from django.db.models import Q
+
+from content_sync import api
+from main.management.commands.filter import WebsiteFilterCommand
+from users.models import User
+from websites.constants import (
+    CONTENT_TYPE_COURSE_LIST,
+    CONTENT_TYPE_PAGE,
+    CONTENT_TYPE_RESOURCE_COLLECTION,
+    PUBLISH_STATUS_NOT_STARTED,
+)
+from websites.models import Website, WebsiteContent
+from websites.serializers import WebsiteBasicSerializer, WebsiteContentSerializer
+
+
+class Command(WebsiteFilterCommand):
+    """Unpublish course webpages provided to the command that have no dependencies. If there are dependencies, print the dependencies."""
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+
+        parser.add_argument(
+            "--user",
+            dest="user",
+            help="Email address of user that will unpublish courses",
+            required=True,
+        )
+
+    def handle(self, *args, **options):
+        super().handle(*args, **options)
+        if not self.filter_list:
+            self.stdout.write("Please provide a list of course ids to unpublish.")
+        else:
```

Comment:
Minor nitpick, take or leave: you can add a `return` line if `filter_list` is null/empty and then omit the `else:` indent.

Reply:
Updated in aae71ff0591b74a5b003e18a400de7b510d188c8. Also writing to `stderr` instead.